packages/headless-react/typedoc/documents/hooks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Hooks
+---
+
+# Hooks
+
+This package provides React hooks for controllers defined in the Engine Definition. Hooks allow you to access controller state and methods easily within your React components.
```

Comment:
Do you think you could add a sentence to explain what the alternative would be? Not really to tell people how to implement commerce ssr without hooks, but to discourage them from doing so 😅 I.e., "You would need to do X and Y, which the hooks can do for you."

I think this would help people understand the use of hooks and why reading this page is important 🙂